src/stats.rs

```diff
@@ -77,28 +71,28 @@ impl ProjectStats {
         }
 
         let lowest_time = now - self.config.budgeting_window;
-        self.budget_buckets.iter().any(|b| b.0 >= lowest_time)
+        self.budget_buckets.iter().all(|b| b.0 < lowest_time)
```

Comment:
oopsie :-D
This should be easily caught by a test. mind adding one as well?

Reply:
I fixed the `test_budgeting` test, which was missing a backoff.